pylutron/__init__.py

```diff
@@ -222,13 +225,20 @@ def _parse_button(self, component_xml):
     """Parses a button device that part of a keypad."""
     button_xml = component_xml.find('Button')
     name = button_xml.get('Engraving')
```

Comment:
will this just be missing in the db completely? or is it an empty string? Sorry, I don't have one of these :(

Reply:
It's missing entirely. Example from my configuration:

```xml
<Button Name="Button 18" UUID="2471" ButtonType="SingleSceneRaiseLower" PrimaryActionType="3" SceneSaver="false" CycleDim="false" AllowDoubleTap="true" StopIfMoving="false" Direction="Lower" ProgrammingModelID="3762" LedLogic="4" ReverseLedLogic="false">
```